packages/cli/src/commands/share.ts

```diff
@@ -0,0 +1,12 @@
+import Truffle from '../models/initializer/truffle/Truffle';
+
+export function getContractsList(message, type) {
```

Comment:
Agree on adding these functions to prompt.ts, to avoid having a generic share.ts file, especially since it is related.